### PR TITLE
UXD-430 Refactored stories to use ExampleStory 🐐

### PR DESCRIPTION
**Pull request by mikrotron**
### Purpose 🚀
Remove copies of `<ExampleStory>` and refactor to use common storybook helper.

### Notes ✏️
- Follow up for https://github.com/acl-services/paprika/pull/757
- Added `component` prop for `<ExampleStory>` to fix links to GitHub.
- Refactored stories to use common `<ExampleStory>`, `<StoryHeading>` and `storyParameters`.
- Also refactored stories using `<StoryHeader>` component to use `<ExampleStory>` instead.
- Refactors stories for `<Button>`, `<FormElement>`, `<ResizeDetector>`, `<Collapsible>`, `<CollapsibleText>`, `<DialogActions>`, and `<SidePanel>`.

### Storybook 📕
http://storybooks.highbond-s3.com/UXD-430--refactored-with-examplestory

### Screenshots 📸
Screen Shot 2020-10-23 at 9 45 06 AM
Variations story for the `<CollapsibleText>` with `<ExampleStory>` and `<StoryHeading>` (instead of `<StoryHeader>`)

### References 🔗
https://aclgrc.atlassian.net/browse/UXD-430

### CODE REVIEW

File: packages/Button/stories/examples/Sizes.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import React from "react";
-import { ButtonStory } from "../../Button.stories.styles";
-import Button from "../../../src";
+import Button from "../../src";
 
 export default function CommonButtons() {
   return (
-    <ButtonStory>
+    <>
```
Comment (nahumzs) [inline packages/Button/stories/examples/Sizes.js:6]: minor: not fragment require

File: packages/Button/stories/examples/States.js [hunk 2]
```diff
@@ -1,13 +1,12 @@
 import React from "react";
-import { ButtonStory } from "../../Button.stories.styles";
-import Button from "../../../src";
+import Button from "../../src";
 
 export default function ButtonStates() {
   return (
-    <ButtonStory>
+    <>
```
Comment (nahumzs) [inline packages/Button/stories/examples/States.js:6]: ✂️ chop chop fragments

File: packages/Button/stories/examples/Sizes.js (hunk 1 above)
Comment (mikrotron) [reply packages/Button/stories/examples/Sizes.js:6]: Oh, I still need it, there are multiple `<div>` children.  But looking at it again, I don't like all those `<divs>`... 🤔

**Review by oscarkwan — APPROVED** [review]
LGTM

**Pull request merged**